src/test/java/org/prebid/server/bidder/pubmatic/PubmaticBidderTest.java

```diff
@@ -108,10 +111,16 @@ public void makeHttpRequestsShouldReturnErrorIfBidRequestExtWrapperIsInvalid() {
     }
 
     @Test
-    public void makeHttpRequestsShouldUseWrapperFromBidRequestExtIfPresent() {
+    public void makeHttpRequestsShouldReturnBidRequestExtIfAcatFieldIsValidAndTrimWhitespace() {
```

Comment:
Why do we rename old test and add logic into it, instead of create new one.
Please, create new test case for acat.

Reply:
In this one old test, when I was adding new logic the test fell and I just change the logic

Reply:
My question is: why do we add logic to old test? Test was had its own purpose. If we need to test new functionality, we create new test.